core/modules/system/system.install

```diff
@@ -40,6 +40,22 @@ function system_requirements($phase) {
         'weight' => -9,
       );
     }
+
+    // Report the configuration storage being used.
+    $config_storage_active = config_get_config_storage('active');
+    $config_storage_staging = config_get_config_storage('staging');
+    $backend = array(
+      '@backend' => is_a($config_storage_active, 'ConfigFileStorage') ? t('files') : t('database'),
+    );
+    $config_storage_locations = array(
+      'active' => t('Active configuration: @backend', $backend),
```

Comment:
This isn't how  `t()` works we need to pass it an array with keys that match what's in the string, like...

`t('Active configuration: !backend', array('!backend' => $backend),`

Reply:
@jenlampton this is actually a trick that I have learned from one of your PRs a few years back IIRC. `$backend` is already an array. It has been pre-prepared in the lines earlier, and it is this:
```php
$backend = array(
  '@type' => $type,
  '!backend' => is_a($class, 'ConfigFileStorage') ? t('files') : t('database'),
);
```

This is keeping the `t()` short and neat, and the array keys/values can be seen right above.

Reply:
...well, after addressing feedback, the array evolved to this:
```php
      $replacements = array(
        '@type' => $type,
        '@prefix' => $prefix,
        '@config_location' => $config_location,
      );
```

And since there is nothing to be translated, I'm using `format_string()` now instead - so no double-escaping via `t()`.

Reply:
OK, I see what you mean now @jenlampton ...I've refactored things, but not 100% sure if I'm doing it properly now 🤔 

Would appreciate it if you had a look.